### PR TITLE
clarify .exe file in cloud cli instructions

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr addresses Jerrie's feedback [https://dbt-labs.slack.com/archives/C051TUB7S9W/p1713980891315809], which adds add'l clarification on where to save the .exe file for windows uses.

### CODE REVIEW

**Review by jkumalahchg — COMMENTED** [review]
This looks great. Provides the right level of information

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ Refer to the [FAQs](#faqs) if your operating system runs into path conflicts.
 
 :::info
 
-Advanced users can configure multiple projects to use the same dbt Cloud CLI by placing the executable in the Program Files folder and [adding it to their Windows PATH environment variable](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53).
+Advanced users can configure multiple projects to use the same dbt Cloud CLI by placing the executable file (`.exe`) in the "Program Files" folder, [adding it to their Windows PATH environment variable](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53), and saving it where needed.
```
Comment (nghi-ly) [inline website/docs/docs/cloud/cloud-cli-installation.md]: sugg: construct as a list (vs v long sentence) for better scannability

```suggestion
Advanced users can configure multiple projects to use the same dbt Cloud CLI by:

 1. Placing the executable file (`.exe`) in the "Program Files" folder
 2. [Adding it to their Windows PATH environment variable](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)
 3. Saving it where needed
```

**Review by nghi-ly — COMMENTED** [review]
looks good! left one n/b sugg

**Review by nghi-ly — APPROVED** [review]
looks good! left one n/b sugg

**Pull request merged**